### PR TITLE
kni: nrop: kickoff 4.19 branch

### PR DESCRIPTION
**Pull request by shajmakh**
Main is now poiting to 4.19, and release-4.18 was branched so update the configuration for main and create 4.18 relevant files.

### CODE REVIEW

**Comment by shajmakh** [comment]
/hold 
for u/s updates to be merged first

File: ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-release-4.18.yaml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    - as: must-gather-e2e-test
+      cli: latest
+      commands: make test-must-gather-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    workflow: optional-operators-ci-operator-sdk-aws
+- as: ci-e2e-install-hypershift
```
Comment (shajmakh) [inline ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-release-4.18.yaml:159]: Note: starting 4.18 we need to run hypershift CI, so inheriting that from main file

**Comment by shajmakh** [comment]
/pj-rehearse pull-ci-openshift-kni-numaresources-operator-release-4.18-ci-install-e2e

**Comment by shajmakh** [comment]
must-gather suites failure is legit and a fix will be filed

**Comment by shajmakh** [comment]
needs https://github.com/openshift-kni/numaresources-operator/pull/1115

**Comment by shajmakh** [comment]
/pj-rehearse periodic-ci-openshift-kni-numaresources-operator-release-4.18-ci-must-gather-e2e

**Comment by ffromani** [comment]
/close

replaced by https://github.com/openshift/release/pull/59948 which extends this PR (and preserves authorship!)

**Pull request closed without merging**